doc/nrf/ug_thread_tools.rst

```diff
@@ -37,22 +37,242 @@ Thread Border Router
 Thread Border Router is a specific type of Border Router device that provides connectivity from the IEEE 802.15.4 network to adjacent networks on other physical layers (such as Wi-Fi or Ethernet).
 Border Routers provide services for devices within the IEEE 802.15.4 network, including routing services for off-network operations.
 
+`OpenThread Border Router`_ (OTBR) released by Google is an open-source implementation of the Thread Border Router recommended to use for development purpose.
+
 Typically, a Border Router solution consists of the following parts:
 
 * Application based on the :ref:`thread_architectures_designs_cp_ncp` design or its :ref:`thread_architectures_designs_cp_rcp` variant compatible with the IEEE 802.15.4 standard.
   This application can be implemented, for example, on an nRF52 device.
 * Host-side application, usually implemented on a more powerful device with incorporated Linux-based operating system.
 
-|NCS| does not provide a complete Thread Border Router solution.
+The |NCS| does not provide a complete Thread Border Router solution.
 For development purposes, you can use `OpenThread Border Router`_, an open-source Border Router implementation that you can set up either on your PC using Docker or on Raspberry Pi.
+
 OpenThread Border Router is compatible with Nordic Semiconductor devices.
+It minimally supports the following functions:
+
+* Bidirectional IP connectivity between Thread and Wi-Fi or Ethernet networks (or both).
+* Bidirectional service discovery using mDNS (on Wi-Fi or Ethernet link, or both) and SRP (on Thread network).
+* Thread-over-infrastructure that merges Thread partitions over IP-based links.
+* External Thread commissioning (for example, using a mobile phone) to authenticate and join a Thread device to a Thread network.
+
+You can configure OpenThread Border Router either manually on Raspberry Pi or using a Docker container, as described in the following sections.
+If you go for the Docker container option, you must :ref:`configure radio co-processor <ug_thread_tools_tbr_rcp>` first.
```

Comment:
I agree with @ru-fu. Wouldn't be better to a little bit reorganize this document and place `Configuring radio co-processor` on the first place as pre-requisity for following parts? If this section is kind of duplicated in the OpenThread Codelabs guide, you could place a note like this to make User understand what's going on: "In case of choosing Raspberry Pi option, you will find this section duplicated by the `OpenThread Border Router Codelab tutorial`_ _Build and flash RCP firmware_ section. You can either omit the current section for the Raspberry Pi and follow the Codelab tutorial or do it now and later omit _Build and flash RCP firmware_ section."

Reply:
After F2F discussion, we would always mandate to go with "configure radio co-processor" section and omit the duplicated section in the Codelabs.